logstash-core/spec/logstash/plugin_spec.rb

```diff
@@ -7,6 +7,16 @@
 require "logstash/filters/base"
 require "support/shared_contexts"
 
+class CustomFilterDeprecable < LogStash::Filters::Base
+  config_name "simple_plugin"
+    config :host, :validate => :string
+
+    def register
+      puts "Register invoked"
```

Comment:
nitpick: I don't see assertions against stdout, so I _think_ this stray `puts` can go away.

```suggestion
```